package.json

```diff
@@ -25,11 +25,23 @@
     "stylelint": "stylelint **/*.scss",
     "test": "lerna run test",
     "typecheck": "lerna run typecheck",
-    "prepare": "lerna run prepare",
-    "postprepare": "[ -z \"$CI\" ] && husky install || exit 0"
+    "build": "lerna run build --ignore @okta/odyssey-ie-11-sandbox --ignore @okta/odyssey-storybook",
```

Comment:
we'll need to be careful here because in our release script we *do* want `@okta/odyssey-storybook` to be built. I think we should just remove the `build` script for these two ignored packages instead and allow them to be constructed in another way. no reason to have a static build for `ie-11-sandbox` that can just be deleted and for `@okta/odyssey-storybook` we can trigger it explicitly in our release script in a separate follow up PR - please make a note for that work!

Reply:
Update: Made all `prepack` scripts run `yarn build` to ensure packages are built before publishing

Reply:
lets remove that `prepack` build script as it will run duplicate builds in our release script. Instead we'll want to build after yarn install normally like we do already (ignoring ie11 and storybook) then explicitly also build storybook: `yarn 
 workspace @okta/odyssey-storybook build` or similar